change/@fluentui-react-examples-85f99e10-eae9-4fdc-8aac-7996bca1bc35.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "chore(react-examples): process collocated stories properly and remove react-menu from dependencies",
+  "packageName": "@fluentui/react-examples",
+  "email": "martinhochel@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
not really sure about the type, I'd say this is a breaking change hmm, but from what I remember, this package is not being used by anyone ?